daqWheel/.gitignore

```diff
@@ -0,0 +1,5 @@
+.pio
```

Comment:
DAQ wheel should be unchanged or you will run into merge conflicts I think... try to see if you can revert the daqWheel modifications